modyn/selector/internal/selector_strategies/presampling_strategies/utils.py

```diff
@@ -0,0 +1,31 @@
+from modyn.selector.internal.selector_strategies.presampling_strategies import AbstractPresamplingStrategy
+from modyn.utils import dynamic_module_import
+
+
+def instantiate_presampler(
```

Comment:
i think we can write some helper function `instantiate_class` like Robin wrote for functions in PR #269: https://github.com/eth-easl/modyn/blob/131ee591930b1b87b83e5bc57e508211d108737a/modyn/utils/utils.py#L170C2-L170C2

Reply:
(because right now we do this in several places, and esp. here with downsamplers it is quite duplicated)

Reply:
I'm not a big fan of exec since it's one the unsafest operation you can do. Technically, you can do something like
`deserialize_function("import shutil; shutil.rmtree("/");")` and everything is gone. I think that instantiate_{presampler/downsampler} requires some knowledge on the presampler/downsampler class so it's difficult to abstract and rely on a single function. I think I'll be able to abstract no more than 50% of the function

Reply:
Sorry, I don't understand your point here. `exec` would not be required. What I meant is implementing a helper function `instantiate_class` (similar to Robin's `instantiate_function` which requires exec, but _class wouldn't). So basically having the entire `hasattr`/`getattr` stuff in one class (since this pattern is used in many parts in Modyn)

Reply:
ok done